performance/sse/test_sse_connection.py

```
@@ -0,0 +1,55 @@
+SSE_ENDPOINT = os.getenv("SSE_ENDPOINT")
+
+USER_COUNT = 50
+DURATION = 600  # 10분
+PING_TIMEOUT = 20  # 15초 ping 기준 + 여유
+
+
+async def listen_to_sse(user_id, results):
+    print(f"[User {user_id}] 연결 시작")
+    try:
+        async with aiohttp.ClientSession() as session:
+            async with session.get(SSE_ENDPOINT, timeout=None) as resp:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)